tests/common/include/tests/common/core_detector.inl

```diff
@@ -661,7 +661,7 @@ TEST(detector, detector_volume_construction) {
         {mask_id::e_rectangle2, 2u},
         {mask_id::e_rectangle2, 3u},
         {mask_id::e_trapezoid2, 1u},
-        {mask_id::e_cylinder2, 2u},
+        {mask_id::e_cylinder2, 0u},
```

Comment:
and here as well

Reply:
There is now only one surface in the mask collection referred to by ```e_cylinder2```, instead of three. Therefore, the mask that has the ```e_cylinder2``` type id now live at the beginning of its own collection, instead of being appended to the portal cylinders

Reply:
OK